### ISSUE TITLE
Range Input - WCAG Rule 1.4.11 Non-text Contrast

### ISSUE THREAD

**Issue by KennyAtHPE** [grommet/grommet-theme-hpe#461]
- Filled portion of the track vs non filled portion fails contrast ratio requirements when range input is placed on top of background default, front, and back.
- Filled portion of the track fails contrast ratio requirements when range input is placed on top of background back.
- (Dark mode) Thumb vs non filled portion of the track fails contrast ratio requirements when range input is placed on top of background default, back, and front
- (Dark mode) Focus indicator vs non filled portion of the track fails contrast ratio when range input is placed on top of background default, back, and front
- (Dark mode) Focus indicator fails contrast ratio when range input is placed on top of background default, back, and front

**Comment by jcfilben** [comment]
Confirm that the latest token updates have resolved this issue

**Comment by jcfilben** [comment]
I checked with the latest token colors and here were the results:

### Light mode

#### B=background-front & background
Pass - green vs grey
Pass - green vs background
Fail - grey vs background

#### background-back
Pass - green vs grey
Pass - green vs background
Fail - grey vs background

### Dark Mode

#### background-front
Fail - green vs grey
Pass - green vs background
Fail - grey vs background

#### background & background-back
Fail - green vs grey
Pass - green vs background
Fail - grey vs background

**Comment by taysea** [comment]
Reopening until we release in grommet-theme-hpe and update docs.
